go/engine/signup.go

```diff
@@ -238,14 +238,16 @@ func (s *SignupEngine) registerDevice(a libkb.LoginContext, ctx *Context, device
 	// change during the signup process.
 	if s.arg.StoreSecret {
 		secretStore := libkb.NewSecretStore(s.G(), s.me.GetNormalizedName())
-		secret, err := s.lks.GetSecret(a)
-		if err != nil {
-			return err
-		}
-		// Ignore any errors storing the secret.
-		storeSecretErr := secretStore.StoreSecret(secret)
-		if storeSecretErr != nil {
-			s.G().Log.Warning("StoreSecret error: %s", storeSecretErr)
+		if secretStore != nil {
```

Comment:
Yes, except in tests.  

I first just set it to true here in the engine, but it caused all kinds of test failures.  The tests are old and don't matter anymore, but it looked hairy to detangle them all and didn't want to spend any time on it.